libvips/foreign/dzsave.c

```diff
@@ -248,9 +248,30 @@ gsf_output_target_write( GsfOutput *output,
 static gboolean
 gsf_output_target_seek( GsfOutput *output, gsf_off_t offset, GSeekType whence )
 {
+#ifdef HAVE_GSF_ZIP64
+	/* No seek needed.
+	 */
+	return( FALSE );
+#else
+	GsfOutputTarget *output_target = (GsfOutputTarget *) output;
+	int stdio_whence;
+
+	switch( whence ) {
+	case G_SEEK_CUR: stdio_whence = SEEK_CUR; break;
+	case G_SEEK_END: stdio_whence = SEEK_END; break;
+	case G_SEEK_SET: stdio_whence = SEEK_SET; break;
```

Comment:
I looked up `G_SEEK_SET` and they don't use the standard values! I agree this switch is crazy, but sadly necessary :(